src/q8/sigproc-ELEC2900/exam/2019/Juin/All/sigproc-ELEC2900-exam-2019-Juin-All.tex

```diff
@@ -0,0 +1,107 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\usepackage{../../../../../../eplcode}
+
+\hypertitle{Signal processing}{8}{ELEC}{2900}{2019}{Juin}{All}
+{Olivier Leblanc \and Julien Verecken}
+{Luc Vandendorpe, Benoît Macq and Laurent Jacques}
+
+Exam during 4h with 6 sheets of questions: 2 questions of LV, 3 questions of LJ et one bonus question about the Python sessions.
+
+\section{Questions of Luc Vandendorpe}
+
+\subsection{LV1}
+
+We are given a signal $x_5[n]$ sampled at 5Ms/s, grouped into blocks of length $N$. We would like to produce the signal $x_6[n]$ at 6Ms/s and use the DFT operator in this aim.
+\begin{enumerate}
+    \item Detail the steps necessary to produce the signal $x_6[n]$.
+    \item Discuss the impact of $N$.
+\end{enumerate}
+We are now interested in the complementary operation which is the conversion of a signal $y_6[n]$ at 6Ms/s into a signal $y_5[n]$ at 5Ms/s.
+\begin{enumerate}
+    \item Detail the steps necessary to produce the signal $y_5[n]$.
+    \item Discuss the impact of $N$.
+\end{enumerate}
+
```

Comment:
```suggestion
\nosolution
```

I know you separated this for each teacher but then potential solutions become very long. In my opinion, the cleaner option is to simply add a solution (or lack thereof) after every question.